wiki/People/The_Team/Nomination_Assessment_Team/th.md

```diff
@@ -1,30 +1,28 @@
 ---
 tags:
   - NAT
-outdated: true
-outdated_since: a04ec9bd266f59392ab0bbc214f699995c4da97c
 ---
 
 # Nomination Assessment Team
 
 **Nomination Assessment Team** (***NAT***) เป็นผู้ดูแลของกลุ่ม [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BN*) และทำให้รับรองว่าฝั่งการทำบีทแมพของ osu! ทำงานอย่างราบรื่น
 
-สมาชิกของ NAT ถูกจำแนกโดยชื่อสีแดงและตำแหน่งยศของพวกเขา เช่นเดียวกับ [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) พวกเขามีสิทธิ์ในการดูแลกรั่นกรองทั่วเว็บไซต์ และมีชื่อสีแดงเข้มในช่องแชทในเกม
+สมาชิกของ NAT ถูกจำแนกโดยชื่อสีแดงและตำแหน่งยศของพวกเขา เช่นเดียวกับ [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) (*GMT*) พวกเขามีสิทธิ์ในการดูแลกลั่นกรองทั่วเว็บไซต์ และมีชื่อสีแดงเข้มในช่องแชทในเกม
 
-**สมาชิกทุกคนใน Nomination Assessment Team ได้สาบานที่จะทำตาม [จรรยาบรรณผู้ช่วยเหลือ](/wiki/Contributor_Code_of_Conduct) นอกเหนือจาก[กฎชุมชน](/wiki/Rules)**
-
-หากคุณพบเห็นสมาชิกคนใดของ Nomination Assessment Team มีพฤติกรรมที่ไม่เหมาะสม โปรดติดต่อ [abuse@ppy.sh](mailto:abuse@ppy.sh) โดยทันที พร้อมข้อมูลมากที่สุดเท่าที่เป็นไปได้ สำหรับรายละเอียดเพิ่มเติม ดูที่ [รายงานการละเมิด](/wiki/Reporting_Bad_Behaviour/Abuse)
+**สมาชิกทุกคนใน Nomination Assessment Team ได้สาบานที่จะทำตาม[จรรยาบรรณผู้ช่วยเหลือ](/wiki/Contributor_Code_of_Conduct) นอกเหนือจาก[กฎชุมชน](/wiki/Rules)**
```

Comment:
```suggestion
**สมาชิกทุกคนใน Nomination Assessment Team ได้สาบานที่จะทำตาม [จรรยาบรรณผู้ช่วยเหลือ](/wiki/Contributor_Code_of_Conduct) นอกเหนือจาก[กฎชุมชน](/wiki/Rules)**
```

Reply:
don't think I agree, we should stick to using spaces as if the hyperlink doesn't exist. this also applies to Thai Wikipedia, I believe

Reply:
ok then